medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -43,3 +43,8 @@ def name_search(self, name='', args=None, operator='ilike', limit=100):
 
         recs = self.search(domain + args, limit=limit)
         return recs.name_get()
+
+    @api.onchange('medical_medication_id')
+    def _onchange_medical_medication_id(self):
+        self.dispense_uom_id = \
```

Comment:
Also, you would be calling it on the rx line in the context of the test. `self.rx_line.method()`